boards/st/stm32n6570_dk/Kconfig.defconfig

```diff
@@ -0,0 +1,10 @@
+# STM32N6570_DK Nucleo board configuration
+
+# Copyright (c) 2024 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_STM32N6570_DK
```

Comment:
all comments in other commit apply here too

Reply:
Done